lib/topological_inventory/sync/inventory_upload/payload/cfme.rb

```diff
@@ -19,10 +19,14 @@ def process
 
                 find_or_create_application(source_type, source)
 
-                inventory = Parser::Cfme.parse(source_type, source_uid, source_name, provider_payload)
-                send_to_ingress_api(inventory)
-
-                logger.info("Processing CFME Provider [#{source_uid}] [#{source_name}]...Complete")
+                if source_exists_in_topology_inventory?(source_uid)
+                  inventory = Parser::Cfme.parse(source_type, source_uid, source_name, provider_payload)
+                  send_to_ingress_api(inventory)
+
+                  logger.info("Processing CFME Provider [#{source_uid}] [#{source_name}]...Complete")
+                else
+                  logger.error("Processing CFME Provider [#{source_uid}] [#{source_name}]...Failed")
```

Comment:
I would like text "...not found in Topological Inventory...Failed" or something more detailed.